client/state/sites/selectors.js

```diff
@@ -3,12 +3,13 @@
 /**
  * External dependencies
  */
-import { map, filter, some, includes, find, get } from 'lodash';
+import { camelCase, map, mapKeys, mapValues, filter, some, includes, find, get } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import createSelector from 'lib/create-selector';
+import { rawToNative as seoTitleFromRaw } from 'components/seo/meta-title-editor/mappings';
```

Comment:
I prefer to leave it here because there's no need for the extra specificity within the `mappings.js` file - the beauty of namespacing :smile: